### ISSUE TITLE
[Error] Too Few Arguments (v.2.1.2)

### ISSUE THREAD

**Issue by DuTek** [nishad/udemy-dl#195]
Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION].

Replace everything between stars with current version of your udemy-dl (output of udemy-dl -v):  
Version : **X.X.X**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [x ] I have read and understood the [contributors guide](https://github.com/nishad/udemy-dl/blob/master/.github/CONTRIBUTING.md).
- [ x] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [x ] I have checked that the issue I'm posting isn't already reported.
- [x ] I have checked that the issue I'm porting isn't already solved and no duplicates exist in [closed issues](https://github.com/nishad/udemy-dl/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/nishad/udemy-dl/issues)
- [x ] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one, eg [X]):**

- [ ] I have never used the project.
- [ x] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

**If the project maintainer has any additional requirements, they will be listed here.**

- No additional requirements.

---
Basically right now everytime I open it: It gives me "Too few arguments". I've checked other issues and found nothing. Also, I have used it before in this same computer and it worked, no idea what's going on.
![udemydl error too few arguments](https://cloud.githubusercontent.com/assets/26068276/23496495/6e23b646-ff16-11e6-80a4-0fbde51c22cf.png)

**Comment by DuTek** [comment]
Absolutely no comments. Forgot how to use command line. Early in the morning. Sorry people
